content/en/shokometadata/install.md

```diff
@@ -59,14 +62,106 @@ Move both of these folders into the following directory.
 
 Proceed to **Editing Plex Scanners** to make the required edits for the scanners to work. 
 
-##### Linux
+### Linux
+
+**Before you continue**, to make it easier to follow the guide, regardless of the Plex install method, we will assign an environment
+variable (`$PLEX_HOME`) that will point to the base directory for your Plex content.
+
+Below are the three most common install methods, and what you need to do for each one before you continue with the rest of the guide. If you
+installed Plex using another method, then please assign the variable to your own, and be sure to use the correct user when executing the
+steps listed below.
+
+##### Bare-metal install
+
+Sign in with **the user responsible for running PMS** (be sure to replace `$USERNAME` with the username or user-id),
+```sh
+sudo su --login --shell=/bin/bash $USERNAME
+```
+
+and assign thes variable.
+```sh
+export PLEX_HOME="/var/lib/plex/Plex Media Server"
+```
+
+Skip to **The steps**.
+##### Snap (plexmediaserver)
+
+Sign in as `root` on your host-system,
+```sh
+sudo su --login --shell=/bin/bash root
```

Comment:
I know this is merged, but any reason for doing this, in comparison to `sudo -i` or, if you need a user `sudo -iu <user>`

I've always hated the thought of using `sudo su` as you're literally using a setuid binary to setuid again

Reply:
An alternative is `sudo -s`
I'd have to check the possible caveat with `sudo -i` when using a user who's shell is `/bin/false` but  `-s` seems like that might work around it

Reply:
Yeah. And , instead of assuming that the user's default shell is anything useful (if they have a shell set at all), I'm assuming that they have `/bin/bash` is installed. It can easily be changed to plain `/bin/sh`, since the commands don't require any bash-spesific features.

So, the final command would be `sudo --login --shell /bin/sh` or `sudo -i -s /bin/sh` for **root**, and `sudo --login --shell /bin/sh --user $USERNAME` or `sudo -i -s /bin/sh -u $USERNAME` for the user that is mentioned in the `$USERNAME` environment variable.

Reply:
I (re-)found the reason i use `sudo su`, because you can't login _and_ change the shell using only `sudo`. You could just change the shell, but then you're still in the same directory, and the user won't create a login session. I think it's ok to only change the shell in this situation, so `sudo -s /bin/bash (-u $USERNAME)` will work.